Library/Formula/bazel.rb

```diff
@@ -0,0 +1,53 @@
+class Bazel < Formula
+  desc "Google's own build tool"
+  homepage "http://bazel.io/"
+  url "https://github.com/bazelbuild/bazel/archive/0.1.1.tar.gz"
+  sha256 "49d11d467cf9e32dea618727198592577fbe76ff2e59217c53e3515ddf61cd95"
+
+  depends_on :java => "1.8+"
+
+  def install
+    system "./compile.sh"
+
+    (prefix/"base_workspace").mkdir
+    cp_r Dir["base_workspace/*"], (prefix/"base_workspace"), :dereference_root => true
+    bin.install "output/bazel" => "bazel"
+    (prefix+"etc/bazel.bazelrc").write <<-EOS
+build --package_path=%workspace%:#{prefix}/base_workspace
```

Comment:
Nit, but can this be indented two spaces after the initial bracket above? You can use `EOS.undent`. Just looks a bit neater.